emscripten.py

```diff
@@ -452,8 +338,202 @@ def move_preasm(m):
       logging.debug('  emscript: python processing: function tables and exports took %s seconds' % (time.time() - t))
 
     return (post, funcs_js, sending, receiving, asm_setup, the_global, asm_global_vars,
-            asm_global_funcs, pre_tables, final_function_tables, exports,
-            function_table_data, forwarded_json)
+            asm_global_funcs, pre_tables, final_function_tables, exports, function_table_data)
+
+
+def finalize_funcs_js(base_funcs_js, asm_setup, the_global, sending, receiving, asm_global_vars,
```

Comment:
Turns out `funcs_js` is many things, and what it is depends on where we are in execution. Renamed it as it transforms to hopefully be more clear.